app/src/store/modules/features.js

```diff
@@ -306,22 +306,24 @@ const actions = {
             Object.entries(pM).forEach(([key, value]) => {
               if (Object.prototype.hasOwnProperty.call(data[rr], value)) {
                 if (key === 'subAoi') {
-                  // dummy empty geometry
-                  let ftrs = [];
                   try {
                     // assuming sub-aoi does not change over time
                     if (!['', '/'].includes(data[rr][value])) {
```

Comment:
This is a possibly dangerous cut into the data structure, but i could not find any other use of the subAoi than in the map.
The reason for this change was to unify the data structure to go along `src/config/trilateral.js`, as the data there are real GeoJSON, in contrast to an array containing (always?) a single GeoJSON here. Please correct me if I am missing something.

Reply:
This is probably something we need @lubojr for to answer :thinking:

Reply:
Sorry, I missed notification on the comment.
Yes `subaoi` is only used in the map component.

And yes, it can be simplified to use a single geojson feature instead of feature collection with 1 feature, as there are never more subaoi features than 1 (and current code would anyway not parse it from source data.

Currently there are a few places in IndicatorMap where code expects a feature collection with 1 feature and breaks the map if format is changed. I have quickly tried to identify the places:
[subaoi_patch.diff.txt](https://github.com/eurodatacube/eodash/files/8531391/subaoi_patch.diff.txt)

Reply:
ok, as long as only the `indicatorMap` needs this format, i think this should be fine, as those places need to be redone (or have already been redone)